Testscripts/Linux/ETHTOOL-OFFLOADING-SETTING.sh

```diff
@@ -133,16 +133,16 @@ if [ $? -ne 0 ]; then
 fi
 
 # Install iPerf3
-ssh -i "$HOME"/.ssh/"$SSH_PRIVATE_KEY" -o StrictHostKeyChecking=no "$remote_user"@"$VF_IP2" ". /home/${SUDO_USER}/utils.sh && update_repos && install_package iperf3"
+ssh -i "$HOME"/.ssh/"$SSH_PRIVATE_KEY" -o StrictHostKeyChecking=no "$remote_user"@"$VF_IP2" ". /home/${SUDO_USER}/utils.sh && update_repos && install_package iperf3 && stop_firewall"
 if [ $? -ne 0 ]; then
     LogErr "Could not install iPerf3 on VM2 (VF_IP: ${VF_IP2})"
     SetTestStateFailed
     exit 0
 fi
 
 update_repos
-install_package iperf3
```

Comment:
Could we updated install_package, log message about redirect to installing_iperf3, when iperf3 in the only one input param

Reply:
in install_package, it will invoke different functions yum_install/apt_get_install/zypper_install etc. If so, I need update in every function?